libraries/testkit/cluster.py

```diff
@@ -141,7 +141,15 @@ def reset(self, sg_config_path):
             "autoimport": "",
             "xattrs": ""
         }
-
+        if self.cbs_ssl and get_sg_version(self._cluster_config) >= "1.5.0":
```

Comment:
This is a string comparison. compare_versions(version_one, version_two): in keywords.utils might be a better fit.